ReferencePluginE/packages.config

```diff
@@ -1,5 +1,5 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="ParatextEmbeddedUiPluginInterfaces" version="2.0.9" targetFramework="net48" />
-  <package id="ParatextPluginInterfaces" version="2.0.17" targetFramework="net48" />
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="ParatextEmbeddedUiPluginInterfaces" version="2.0.21" targetFramework="net48" />
+  <package id="ParatextPluginInterfaces" version="2.0.21" targetFramework="net48" />
```

Comment:
Ideally, ReferencePluginE should stay being 2.0.9 and a new plugin should be created to access the new stuff. I'm not sure it's worth changing at this point, but doing it this way means that we aren't sure if we broke any of the other stuff this reference plugin tests from 2.0.9 to 2.0.21.